monitor/events.go

```diff
@@ -0,0 +1,89 @@
+package monitor
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"time"
+
+	"github.com/golang/glog"
+)
+
+type Event struct {
+	Name       string            `json:"event"`
+	Properties map[string]string `json:"properties"`
+}
+
+var EventsURL = "http://metrics.livepeer.org/events"
+
+func LogStreamCreatedEvent(manifestID string, createTime time.Time) {
+	e := &Event{
+		Name: "StreamCreated",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"createTime": fmt.Sprintf(createTime.Format(time.UnixDate)),
+		},
+	}
+
+	sendPost(e)
+}
+
+func LogStreamStartedEvent(manifestID string, startTime time.Time) {
+	e := &Event{
+		Name: "StreamStarted",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"startTime":  fmt.Sprintf(startTime.Format(time.UnixDate)),
+		},
+	}
+
+	sendPost(e)
+}
+
+func LogStreamEndedEvent(manifestID string, createTime time.Time, endTime time.Time) {
+	e := &Event{
+		Name: "StreamEnded",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"createTime": fmt.Sprintf(createTime.Format(time.UnixDate)),
+			"endTime":    fmt.Sprintf(endTime.Format(time.UnixDate)),
+		},
+	}
+
+	sendPost(e)
+}
+
+func LogStreamCreateFailed(manifestID string, reason string) {
+	glog.Infof("Logging StreamCreateFailed...")
+	e := &Event{
+		Name: "StreamCreateFailed",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"reason":     reason,
```

Comment:
Since we're putting in timestamps for other events, maybe add one here as well?

Might it also be good to have a consistent naming for the timestamp field, eg `loggedAt` or something, rather than `createTime`/`startTime`/`endTime` etc? That way we can have a generic bit of code for scanning over various types of events with a common naming structure.

Reply:
Yeah the timestamp will be automatically added when the daemon inserts the event into the db.